public/bellomos.html

```diff
@@ -1,41 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
-    <link
-      rel="stylesheet"
-      href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"
-      integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A=="
-      crossorigin=""
-    />
-    <script
-      src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
-      integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA=="
-      crossorigin=""
-    ></script>
-    <link href="index.css" rel="stylesheet" />
-  </head>
-  <body>
-    <div class="Navbar">
-      <a href="ramuntos.html">Ramunto's</a>
-      <a href="johnny-boys.html">Johnny Boy's Pancake House</a>
-      <a href="orientaste.html">Orientaste</a>
-      <a href="bellomos.html">Bellomo's Market</a>
-      <a href="rollin-rooster.html">The Rollin' Rooster</a>
-      <a href="taco-bell.html">Taco Bell</a>
-    </div>
```

Comment:
We could generate this through a fetch to our /api and use a loop that for each of the names we get back, we generate a link with the name in it and the route it would navigate to.